src/common/ipfixlolib/ipfixlolib.h

```diff
@@ -125,9 +125,16 @@ See ipfixlolib.h for details on how to use this library.
 extern "C" {
 #endif
 
+/* Supported export protocols */
+typedef enum export_protocol_version {
```

Comment:
Would it be possible to combine the two defines `define NFV9_VERSION_NUMBER 0x0009` and `IPFIX_VERSION_NUMBER 0x000a` and the `enum export_protocol_version`?